### PR TITLE
Pre release version0

### PR DESCRIPTION
**Pull request by gx-ai-architect**
Addressed the following issues:
1. https://github.com/Red-Hat-AI-Innovation-Team/reward_hub/issues/10
2. https://github.com/Red-Hat-AI-Innovation-Team/reward_hub/issues/9

### CODE REVIEW

File: reward_hub/hf/reward.py [hunk 1]
```diff
@@ -91,11 +91,13 @@ def score(self, question: str, responses: List[str], max_token_length: int = 819
                     return_tensors="pt",
                     padding=True,
                     truncation=True,
-                    max_length=max_token_length,
+                    max_length=max_input_tokens,
                 ).to(self.model.device)
                 with torch.no_grad():
                     output = self.model(input_ids)
-                    reward_score = output.score.float().item()
+                    # The preference score for the response, aggregated from the
```
Comment (xukai92) [inline reward_hub/hf/reward.py:98]: what does this comment mean/do here?

File: reward_hub/hf/reward.py [hunk 2]
```diff
@@ -212,3 +211,26 @@ def score(self, question: str, responses: List[str], step_separator: str = "\n\n
             return [PRMResult(scores=scores, aggregate_method=aggregate_method) for scores in all_scores]
         else:
             return [PRMResult(scores=scores, aggregate_method=aggregate_method).score for scores in all_scores]
+
+if __name__ == "__main__":
```
Comment (xukai92) [inline reward_hub/hf/reward.py:215]: do we really want these script functions, or they should be moved to `tests` or a dedicated file in `scripts`

File: reward_hub/vllm/reward.py [hunk 3]
```diff
@@ -73,26 +74,30 @@ def score(self, question: str, responses: List[str], step_separator: str = "\n\n
...
             raise ValueError(f"Model {self.model_name} is not supported")
         
         if return_full_prm_result:
-            return [PRMResult(scores=scores, aggregate_method=aggregate_method) for scores in all_scores]
+            return [PRMResult(scores=scores) for scores in all_scores]
         else:
             return [PRMResult(scores=scores, aggregate_method=aggregate_method).score for scores in all_scores]
+
+
+if __name__ == "__main__":
```
Comment (xukai92) [inline reward_hub/vllm/reward.py:97]: same as above. please revise all

File: reward_hub/hf/reward.py (hunk 2 above)
Comment (gx-ai-architect) [reply reward_hub/hf/reward.py:215]: they should be moved. I agree

**Comment by gx-ai-architect** [comment]
Merge as the raised issue is addressed in following PRs: https://github.com/Red-Hat-AI-Innovation-Team/reward_hub/pull/15
